FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+State.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ComposeViewController+State.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 4/6/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - State Handling
+extension ComposeViewController {
+    internal func updateState(with newState: State) {
+        if case .searchEmails = newState, !self.shouldDisplaySearchResult {
+            return
+        }
+
```

Comment:
This would belong in `RecipientInput`, probably.

Reply:
But this function updates recipient list, password section, search results.

Reply:
I see. We can leave it as is for now. Later, maybe it needs some renaming to make it clearer what it does.

Reply:
Ok